corehq/apps/hqadmin/reporting/reports.py

```diff
@@ -294,13 +301,7 @@ def get_active_users_data(domains, datespan, interval, datefield='date',
                 .size(0)
                 .run()
                 .facets.user.result
-                if u['term'] in (
-                    UserES()
-                    .show_inactive()
-                    .mobile_users()
-                    .domain(domains)
-                    .run().doc_ids
-                )
+                if u['term'] in mobile_users
```

Comment:
Looks like it is the same id https://github.com/dimagi/commcare-hq/blob/fe89120f5c25d3b03bde521d0712a61f7c083af3/corehq/apps/es/forms.py#L23